src/app/lib/styledGrid/index.stories.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import styled from 'styled-components';
 import { storiesOf } from '@storybook/react';
 import {
-  GhostWrapper,
+  GhostGrid,
```

Comment:
This Story file can use `Grid` throughout.

Reply:
done :)